runtime/functions

```diff
@@ -378,16 +442,15 @@ configure_postgresql() {
 
   # start postgres server internally for the creation of users and databases
   rm -rf ${PG_DATADIR}/postmaster.pid
-  set_postgresql_param "listen_addresses" "127.0.0.1" quiet
-  exec_as_postgres ${PG_BINDIR}/pg_ctl -D ${PG_DATADIR} -w start >/dev/null
+  start_postgres_daemon
 
   create_user
   create_database
   create_replication_user
 
+  run_startup_scripts
+
   # stop the postgres server
-  exec_as_postgres ${PG_BINDIR}/pg_ctl -D ${PG_DATADIR} -w stop >/dev/null
+  stop_postgres_daemon
 
-  # listen on all interfaces
-  set_postgresql_param "listen_addresses" "*" quiet
```

Comment:
I'll have to check now, this was two months ago.

I believe I had an issue with this line because Postgres starts initially to do the basic import, stops and then restarts.

I believe it was unnecessary because I've also removed
`set_postgresql_param "listen_addresses" "127.0.0.1" quiet`